bike_kollective/lib/authentication_screen.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:bike_kollective/data/model/user.dart';
+import 'package:bike_kollective/data/provider/database.dart';
+import 'package:bike_kollective/data/provider/active_user.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+
+class AuthenticationScreen extends StatelessWidget {

```

Comment:
This might need to be a ConsumerWidget so it can get a ref to the active user provider